### ISSUE TITLE
Misspecified keyword in the protein pretraining and prediction tutorial

### ISSUE THREAD

**Issue by riversdark** [PaddlePaddle/PaddleHelix#148]
I'm looking at the TAPE [protein pretraining tutorial](https://github.com/PaddlePaddle/PaddleHelix/blob/dev/tutorials/protein_pretrain_and_property_prediction_tutorial.ipynb), in the model config dict there is a keyword `layer_num`, but there is no such keyword argument in any of the TAPE models, only a `n_layers` argument. So whatever `layer_num` hidden layers we specify, since the models only search for the number of hidden layers with the `n_layers` keyword, and since there is no such keyword in `model_config`, it will always use the default value `8`. Am I reading this right?

TLDR: I think the `layer_num` keyword in model configs should be `n_layers`.

**Comment by linxd5** [comment]
hi, riversdark. All you say is right. In the current released version, we have not opened the config function fully, you need to revise the code directly if you want to change the config. Sorry for the bad use experience. We will complete the config function in the next released version. Thanks for your issue.

**Comment by riversdark** [comment]
Hey no need to apologize and thanks for all your great work! I guess I can close this now, or should I wait for you to do it?
